src/main/java/core/basesyntax/model/Report.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.model;
+
+import java.util.Map;
+
+public class Report {
```

Comment:
According to the checklist, you shouldn't begin class implementation with an empty line. Please remove the empty line before the class declaration.